FrenetOptimalTrajectory/fot.py

```diff
@@ -3,9 +3,11 @@
 import numpy as np
 import matplotlib.pyplot as plt
 import matplotlib.patches as patch
+import sys, getopt
+from pathlib import Path
 
 
-def main():
+def run_fot(show_animation=False, show_info=False, thread_num = 0, save_frame=False):
```

Comment:
```suggestion
def run_fot(show_animation=False, show_info=False, num_threads=0, save_frame=False):
```

Reply:
Also, could you rename the variable to `num_threads`?

Reply:
Sure thing.